packages/www/pages/dashboard/developers/webhooks/[id].tsx

```diff
@@ -217,6 +217,34 @@ const WebhookDetail = () => {
                     </Badge>
                   ))}
                 </Cell>
+                <Cell variant="gray">Last trigger</Cell>
+                <Cell>
+                  {data.status
+                    ? format(
+                        data.status?.lastTriggeredAt,
+                        "MMMM dd, yyyy h:mm:ss a"
+                      )
+                    : "Never"}
+                </Cell>
+                <Cell variant="gray">Last failure</Cell>
+                <Cell>
+                  {!data.status
+                    ? "Never"
+                    : data.status.lastFailure
+                    ? format(
+                        data.status.lastFailure.timestamp,
+                        "MMMM dd, yyyy h:mm:ss a"
+                      )
+                    : "Never"}
+                </Cell>
+                <Cell variant="gray"></Cell>
+                <Cell>
+                  {data.status?.lastFailure
+                    ? data.status.lastFailure.response
+                      ? `Status code: ${data.status.lastFailure.statusCode} - ${data.status.lastFailure.response}`
```

Comment:
Hmm, now I think this will look too crammed up on a single line. Do you think it's possible to add a line break between the status code and the response body? 

Also, is that response body displayed as code / with a monospaced font? I wonder if we are being explicit enough about what this is or if it might end up looking like a random string that doesn't make sense to the user.

Here is what I (later) managed to get on my webhook:
![Screen Shot 2022-01-26 at 16 59 28](https://user-images.githubusercontent.com/1613383/151237682-2c858912-2809-476b-b85c-221bc5f15920.png)
The response body is exactly "500 Internal Server Error", with the update here it becomes `500 - 500 Internal Server Error`, none of which seems to be really clear to the user what they actually mean.

Soo... WDYT of adding a row "header" just like we have `Last failure`, `Last trigger` etc above so we are clear about what this means? Could be
 - Last failure: <date>
 - Error response: <status code> <status code text> <new line> <response body>

Or the error response fully separate like
 - Error Status Code: <status code> <status code text>
 - Error Response: <response body> (with proper line wrapping and ideally formatting as code)

WDYT?

Reply:
I think on JavaScript you can get the status text like this:
```
import { STATUS_CODES } from "http";
```
then later
```
STATUS_CODES[status]
```
(not sure how perfectly that will work on the browser tho, gotta test)

Reply:
Good points, the thing I had to consider is that if the request fails, there are no status code and error responses, but just and error message. In any case, I added separate rows and a condition check where if no status code is present, I display `Error message` instead of `Error response`, without the status code row - both monospaced!

This is the result:
![mono](https://user-images.githubusercontent.com/50955448/151364535-b2c6ddad-d63b-4059-8bbd-cccff700c33e.png)

I also added the status text as you suggested, thanks!

Reply:
Looks great!